aepsych/server/server.py

```diff
@@ -799,9 +800,9 @@ def tell(self, outcome, config, model_data=True):
                         param_value=float(param_value),
```

Comment:
Should param_value also be str here?

Reply:
Yes! You're right! 
I think the tests didn't catch it cause there is at least two parameters in all of them